src/Tribe/Attendee_Registration/Template.php

```diff
@@ -20,34 +20,37 @@ class Tribe__Tickets__Attendee_Registration__Template extends Tribe__Templates {
 	public function hook() {
 
 		// Spoof the context
-		add_filter( 'the_posts', array( $this, 'setup_context' ), -10 );
+		add_filter( 'the_posts', [ $this, 'setup_context' ], -10 );
 
 		// Set and remove the required body classes
-		add_action( 'wp', array( $this, 'set_body_classes' ) );
+		add_action( 'wp', [ $this, 'set_body_classes' ] );
```

Comment:
:no_entry_sign: Short array syntax is not allowed

<sub>hash: 2dcad43a9569c737fa27b726c68e0457</sub>